sign/dilithium/internal/common/pack_test.go

```diff
@@ -1,16 +1,19 @@
 package common
 
 import (
-	"math/rand"
+	"crypto/rand"
 	"testing"
 )
 
 func TestPackLe16AgainstGeneric(t *testing.T) {
 	var p Poly
 	var buf1, buf2 [PolyLe16Size]byte
+	pp := make([]uint8, 256)
+
 	for j := 0; j < 1000; j++ {
+		_, _ = rand.Read(pp)
```

Comment:
How about re-reusing randSliceUint32 here?
```go
p = randSliceUint32(256)
for i := range p {
   p[i] &= 0xF
}
```

If you end up passing a max parameter, this is just `p = randUint32Slice(256, 0xF)`.

Reply:
excelent suggestion